config/redhat-certified/validating_webhook.yaml

```
@@ -0,0 +1,8 @@
+apiVersion: admissionregistration.k8s.io/v1
+kind: ValidatingWebhookConfiguration
+metadata:
+  name: validation.konghq.com
+webhooks:
+- name: "gateway-operator-validation.konghq.com"
+  # https://github.com/Kong/gateway-operator/issues/291
```

Maybe just add a TODO so that it's easier to grep for this:

```suggestion
  # TODO: https://github.com/Kong/gateway-operator/issues/291
```